readme.md

```diff
@@ -199,8 +207,26 @@ Each of the generator properties (like `name`, `address`, and `lorem`) are calle
     year($max = 'now')                    // '1993'
     century                               // 'VI'
     timezone                              // 'Europe/Paris'
-
+    
 Methods accepting a `$timezone` argument default to `date_default_timezone_get()`. You can pass a custom timezone string to each method, or define a custom timezone for all time methods at once using `$faker::setDefaultTimezone($timezone)`.
+    
+#### Naming of relative dateTime methods
+In current state we have `dateTimeThisXXX` which is a misleading name. One would assume that this methods
+will return a calendar accurate datetime but rather it will return a random datetime in the "past" month (`$now - 1 month`).
+
+Assuming the current date is `2019-09-02`. Here is a full list of methods and what you can expect from them:
+- `dateTimePastCentury` datetime between `1919-09-02` and `2019-09-02`
```

Comment:
Not sure I have a misunderstanding, but personally, I would expect this method to return a value that corresponds to the past century, that is, the 20th century - so any date between `1900-01-01` and `1900-12-31`.
```

Reply:
You are right. Will change it.

Reply:
Now that I thought about it again. This is the current bahavior. `PastCentury` simply goes back 100 years. So the stated value there is correct.

If you want 1900 to 2000 yiu need to use `LastCentury`.